### PR TITLE
v2.5

### PR DESCRIPTION
**Pull request by TNTPro**

### CODE REVIEW

**Comment by zimbatm** [comment]
@TNTPro do you want to become a maintainer of the project? It seems like you are doing a lot of good work there.

**Comment by TNTPro** [comment]
@zimbatm Hi there.  Yes, you can add me as a maintainer, that would be cool.
You might want to try my latest build & let me know what you think.

File: build.sh [hunk 1]
```diff
@@ -277,37 +461,54 @@ sed -i 's/-lgcc_s/-lgcc_eh/g' x265.pc
...
 cd $BUILD_DIR/libass-*
 [ $rebuild -eq 1 -a -f Makefile ] && make distclean || true
 ./autogen.sh
 ./configure --prefix=$TARGET_DIR --disable-shared
 make -j $jval
 make install
 
-echo "*** Building mp3lame ***"
+echo
+/bin/echo -e "\e[93m*** Building mp3lame ***\e[39m"
```
Comment (zimbatm) [inline build.sh]: you could create a log function to reduce some of that repetition. Eg:
```
log() {
  echo -e "\e[93m*** *@ ***\e[39m"
}

log Building mp3lame
```

**Comment by zimbatm** [comment]
@TNTPro invite sent!

At this point, I think it's good if the project has a new maintainer. Feel free to do what you want with the project.

My recommendations would be to setup Github Actions as a CI to check that things keep working.

**Pull request closed without merging**